var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -20,7 +20,9 @@ class Ucx(AutotoolsPackage, CudaPackage):
     version('1.9-dev', branch='v1.9.x')
```

Comment:
Can you move this to the line above 1.9.0 and then remove `preferred=True` on 1.11.1 since that is redundant?

Reply:
Sure, done!